lib/rules/no-duplicate-imports.js

```diff
@@ -9,87 +9,215 @@
 //------------------------------------------------------------------------------
 
 /**
- * Returns the name of the module imported or re-exported.
+ * Check if (import|export) type is belong to (ImportSpecifier|ExportSpecifier) or (ImportNamespaceSpecifier|ExportNamespaceSpecifier).
+ * @param {string} importExportType An import type to get.
+ * @param {string} type Specifier type can be namespace or specifier
+ * @returns {boolean} True if (import|export) type is belong to (ImportSpecifier|ExportSpecifier) or (ImportNamespaceSpecifier|ExportNamespaceSpecifier) and false if it doesn't.
+ */
+function isImportExportSpecifier(importExportType, type) {
+    const namedTypes = ["ImportSpecifier", "ExportSpecifier"];
```

Comment:
These two arrays can be defined at module level, so they’re not created every time this function runs.

Reply:
Thank you for taking the time to review the PR. I created these two arrays at the module level in section Helpers.